src/cr/cube/dimension.py

```diff
@@ -748,7 +748,7 @@ def __init__(self, element_transforms_dict, prune):
         self._prune = prune
 
     @lazyproperty
-    def fill(self):
+    def fill(self):  # pragma: no cover
```

Comment:
I don't see the justification for a no-cover pragma here. This method does real work and deserves a unit test.